### ISSUE TITLE
Window is not resizable on OSX

### ISSUE THREAD

**Issue by samw3** [HeapsIO/heaps#541]
I'm using Heaps via Hashlink on OSX with hlsdl.  Just using a very basic Hello World example, I noticed that the window is not resizable.  Is this the desired behavior?  Can I somehow set the window to be resizable?

https://www.dropbox.com/s/fhkcssuhwh4tpkb/Screenshot%202019-02-13%2022.05.08.png?dl=0

**Comment by ncannasse** [comment]
@RobertBorghese any idea?

**Comment by RobertBorghese** [comment]
Ack, it appears `haxelib install hldx/hlsdl` still installs the pre-1.9 hashlink package. The old haxe code is being used with the new binaries.

Since the new features merely added features, they seem to almost be compatible. However, as stated here, since the new parameter which passes the SDL_RESIZABLE flag (by default) isn't present in the old code, it isn't being passed as it should in the new implementation. 

Long story short, the haxelib packages need to be updated. Don't know why I didn't realize sooner. 😣

**Comment by ncannasse** [comment]
My fault, the 1.9 SDL+DX libraries have now been pushed to haxelib!
